webpack/assets/javascripts/react_app/components/common/layout/PageTitle.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const PageTitle = ({ text }) => {
+  document.title = __(text);
```

Comment:
I think calling `__()` should be done in `app/views/* rather than here.

Reply:
[Other](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/redux/actions/common/forms.js#L22) js files use it as well. According to [wiki](https://projects.theforeman.org/projects/foreman/wiki/Translating#JavaScript), this is how translating in js is done.

Reply:
although it makes this function less pure (I think using `__()` is a side effect), it is not that bad.
Thanks for pointing that out, though.